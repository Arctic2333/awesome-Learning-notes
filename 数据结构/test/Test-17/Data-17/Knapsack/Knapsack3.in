 4 13
 8 6 4 3
 12 11 9 8
