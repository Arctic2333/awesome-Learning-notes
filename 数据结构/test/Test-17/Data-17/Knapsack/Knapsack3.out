Options: 0 1 1 1 
Max Value: 28
